src/app/shared/table/table.component.html

```diff
@@ -296,9 +297,8 @@
             <ng-container *ngIf="!dataSource.tableDef.rowDetails.angularComponent">
               <td class="col-3em details-column-row"></td>
               <td [attr.colspan]="numberOfColumns-1">
-                <div
-                  [innerHtml]="(dataSource.tableDef.rowDetails ? row[dataSource.tableDef.rowDetails.detailsField] : '')"
-                  class="row-detail">
+                <div class="row-detail">
+                  {{(dataSource.tableDef.rowDetails ? row[dataSource.tableDef.rowDetails.detailsField] : '')}}
```

Comment:
Side effect: the formatting of the detailed logs will be lost as the output is not longer interpreted.
If we set it back to **innerHtml** then in case we have HTML that's in the stack trace (error coming from an external service which is saved in the logs), the HTML will be executed and the output will look like this
![image](https://user-images.githubusercontent.com/23378124/135773468-8c024840-0e19-4f1e-b64d-6abcd2ec0448.png)

Reply:
I would need the formatted logs as I cannot read them anymore:
<img width="1592" alt="2021-10-04_09-51-36" src="https://user-images.githubusercontent.com/25266945/135813633-91cb8566-4e9f-4ce5-9c99-605888188b90.png">

Reply:
indeed this the side effect, we can use the innerHTML but the html could be interpreted as above :(

Reply:
Yes, security sometimes make our life harder.
If you can find a solution to format the text without having security issue, feel free :)

Reply:
the formatting is back, to avoid HTML rendering i'll try to escape these characters "< >"

Reply:
Great, thank you!!!